---
title: C6103
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C6103
ms.assetid: 22d1ab35-31a3-4ba9-8ef4-7a64bce66621
ms.openlocfilehash: f52003bf7bd27390b7ee6332769b0e24226bc364
ms.sourcegitcommit: 7bea0420d0e476287641edeb33a9d5689a98cb98
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 02/17/2020
ms.locfileid: "77420744"
---
# <a name="c6103"></a>C6103

> aviso C6103: retornando \<variável > da chamada de função com falha na linha \<local >.

Um caminho bem-sucedido por meio da função está retornando uma variável usada como um parâmetro `_Out_` para uma chamada de função interna malsucedida.

O problema talvez seja a função chamada e a função de chamada não anotadas por completo. A função de chamada pode exigir `_Always_`, `_Outptr_result_nullonfailure_` (`_COM_Outptr_` para código COM), ou uma anotação relacionada, e a função de chamada pode exigir uma anotação `_Success_`. Outra causa possível para esse aviso é que a anotação `_Success_` na função chamada está incorreta.

## <a name="see-also"></a>Consulte também

[Usando anotações de SAL para reduzir defeitos de código do C/C++](../code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects.md)
